src/Avalonia.Native/WindowImplBase.cs

```diff
@@ -74,6 +76,7 @@ internal abstract class WindowBaseImpl : IWindowBaseImpl,
             _keyboard = AvaloniaLocator.Current.GetService<IKeyboardDevice>();
             _mouse = new MouseDevice();
             _cursorFactory = AvaloniaLocator.Current.GetService<ICursorFactory>();
+            _clipboard = new ClipboardImpl(_factory.CreateClipboard());
```

Comment:
We should still be using a shared instance for platforms where we can do that. ClipboardImpl allocates some native resources, so we don't want to have a separate instance per window.